mlir/utils/jenkins/rock-tests/rock_validate.sh

```diff
@@ -9,7 +9,7 @@ declare -ig TESTFWD=0
 declare -g DTYPE=""
```

Comment:
I think all should stay as MIOpen and this directory contains tests from MIOpen. @jerryyin please confirm